pkgs/by-name/ne/nemo-preview/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  pkgs,
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  meson,
+  pkg-config,
+  ninja,
+  glib,
+  gtk3,
+  cjs,
+  gtksourceview4,
+  gobject-introspection,
+  libmusicbrainz5,
+  webkitgtk_4_1,
+  clutter-gtk,
+  clutter-gst,
+  wrapGAppsHook3,
+  nemo,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nemo-preview";
+  version = "6.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "nemo-extensions";
+    rev = version;
+    hash = "sha256-39hWA4SNuEeaPA6D5mWMHjJDs4hYK7/ZdPkTyskvm5Y=";
+  };
+
+  sourceRoot = "${src.name}/nemo-preview";
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+    meson
+    pkg-config
+    glib
+    ninja
+    (pkgs.xreader.overrideAttrs (
+      finalAttrs: previousAttrs: {
+        mesonFlags = previousAttrs.mesonFlags ++ [
+          "-Dintrospection=true"
```

Comment:
Should this possibly be enabled by default?

Reply:
I was thinking that, but that would be a separate package and a separate PR. We could merge this now and update Xreader afterwards

Reply:
Its not uncommon to update multiple packages in a pr if the changes are connected. Or do you just want this merged quickly?

Reply:
I made the changes to xreader